### PR TITLE
[release] 1.12.1 compiler release note

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit contains a 1.12.1 compiler release note.

Related : #5376 
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Review by chunseoklee — APPROVED** [review]

**Review by hseok-oh — DISMISSED** [review]

**Review by seanshpark — APPROVED** [review]
LGTM

File: docs/release/1.12/release-note-1.12.1.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Release Note 1.9.1
```
Comment (seanshpark) [inline docs/release/1.12/release-note-1.12.1.md:1]: need update :)

**Review by seanshpark — CHANGES_REQUESTED** [review]
need update :)

**Comment by mhs4670go** [comment]
Oh.. terrible mistake

**Comment by mhs4670go** [comment]
@seanshpark @chunseoklee @hseok-oh PTAL!

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by hseok-oh — APPROVED** [review]

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**